### ISSUE TITLE
Logo for Eclipse Ditto

### ISSUE THREAD

**Issue by kartben** [eclipse/ditto#8]
Does Ditto have a logo? If not, let me know if you need my/EF's help setting up a crowd sourcing campaign to get one designed.
It would be nice to have a Ditto logo for the upcoming I4.0 white paper :-)

**Comment by thjaeckle** [comment]
No, we don't have one yet.
Would be great if you could help us with a crowd sourcing campaign. We liked the results of hawkBit and Hono a lot and are very excited to see what comes up for Ditto.

**Comment by thjaeckle** [comment]
@kartben how do we initiate the logo crowd sourcing campaign? I guess we need some texts and explanations and maybe some basic ideas or colors?

**Comment by kartben** [comment]
@thjaeckle apologies, i've been swamped with other stuff… (testbed, white paper… 😊)
Let's sit together sometime next week and just get this done, ok? :-)

**Comment by thjaeckle** [comment]
I started a discussion on the mailing list what the text for the crowdsourcing campaign could be:
https://dev.eclipse.org/mailman/listinfo/ditto-dev

**Comment by thjaeckle** [comment]
Hi @kartben 
We came up with a text for the logo campaign:

---

## Who are you known as?
Ditto

## Tell us a bit about who you are and the people you reach
We need a logo for our open source software project Ditto – [https://projects.eclipse.org/projects/iot.ditto](https://projects.eclipse.org/projects/iot.ditto)

Eclipse Ditto is a technology implementing a software pattern called “Digital Twins” which we would also like to see reflected in the logo. A digital twin is a virtual, cloud based, representation of his real world counterpart (real world “Things”, e.g. devices like sensors, smart heating, connected cars, smart grids, EV charging stations, …). But those twins are not limited to devices, therefore we speak more generically of "Things".<br/>
The technology mirrors potentially millions and billions of digital twins residing in the digital world with physical “Things”. This simplifies working with IoT (Internet of Things) for software developers as they do not need to know how or where exactly the physical “Things” are connected. They can just be used as any other service in the web via their digital twins.

The name of the project has the meaning of the English “ditto”: the same thing again / as has been said before<br/>
The name emphasizes the mirroring / twin aspect as key purpose of the technology.
The Ditto project is part of a larger ecosystem of open-source projects called “Eclipse IoT”. The Eclipse IoT website is available at http://iot.eclipse.org and the logo of Eclipse IoT is attached to the brief – while not mandatory, we encourage designers to provide logos that convey the idea of Ditto belonging to the Eclipse IoT brand.

## What industry do you think your business is most related to?
Technology

## Other notes
* the logo should work both with and without the "Ditto" text

Some important keywords relevant to Ditto that we would like somehow to be represented in our logo:
* Internet of Things
* Cloud
* Scalability (i.e Ditto can give home to billions of digital twins)

We'll need a favicon for our website, in addition to the "regular" logo.

---

How do we proceed now? Can you start a campaign with that information? :blush:

**Comment by kartben** [comment]
@thjaeckle this looks good. I would recommend you create an account on 99designs, create the draft for the campaign, and once everything is ready share the credentials with me and I'll make the payment. Makes sense?

FWIW I recommend going for a "blind" challenge, where the designs are not visible to the other designers, making for better innovation and creativity as people can't steal each others' ideas :-)

**Comment by thjaeckle** [comment]
ok, will do 👍

**Comment by thjaeckle** [comment]
The logo was successfully "found": https://en.99designs.de/logo-design/contests/logo-iot-internet-things-open-source-project-773651

Will upload to Ditto project site and GitHub tomorrow.
